chaoscenter/authentication/api/handlers/grpc/grpc_handler.go

```diff
@@ -0,0 +1,106 @@
+package grpc
+
+import (
+	"context"
+	"fmt"
+	"litmus/litmus-portal/authentication/api/middleware"
+	"litmus/litmus-portal/authentication/api/presenter/protos"
+	"litmus/litmus-portal/authentication/pkg/entities"
+	"litmus/litmus-portal/authentication/pkg/validations"
+	"strconv"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/golang-jwt/jwt"
+)
+
+func (s *ServerGrpc) ValidateRequest(ctx context.Context,
+	inputRequest *protos.ValidationRequest) (*protos.ValidationResponse, error) {
+	token, err := middleware.ValidateToken(inputRequest.Jwt)
+	if err != nil {
+		return &protos.ValidationResponse{Error: err.Error(), IsValid: false}, err
```

Comment:
```suggestion
		return &protos.ValidationResponse{Error: err.Error(), IsValid: true}, err
```
Shouldn't InValid be `true` incase of error?